### ISSUE TITLE
unterminated substitute pattern

### ISSUE THREAD

**Issue by rr0ss0rr** [JayBrown/DNSCrypt-Menu#2]
Hi .. This was working fine at home .. Just arrived at my daughters and I see the dnscrypt icon in the menu but it has a yellow triangle to the right.  If I click, I see a section (below DNSCrypt Menu) that has:
sed: 1: "s/^lo0
2259: //": unterminated substitute pattern

Thanks

**Comment by JayBrown** [comment]
I just uploaded v1.0.28 that should hopefully fix this `sed` error.

Explanation: it concerns the output of the `ifconfig` command, which is used in three places in the plugin, e.g.:

```
❯ ifconfig lo0
lo0: flags=8149<UP,LOOPBACK,RUNNING,PROMISC,MULTICAST> mtu 16384
	options=3<RXCSUM,TXCSUM>
	inet6 ::1 prefixlen 128
	inet 127.0.0.1 netmask 0xff000000
	inet6 fe80::1%lo0 prefixlen 64 scopeid 0x1
	inet 127.0.0.54 netmask 0xff000000
	nd6 options=1<PERFORMNUD>
```

The sed command just replaces the "lo0: " at the beginning of the first line with nothing, because we already have the device name displayed elsewhere. This is done with a variable, let's call it `$var`.

On my system (OS X 10.11.6) it worked without problems with `sed "s/^$var: //"`, "s" meaning replace ("substitute"), and "^" meaning the beginning of the (first) line, but I've now changed it to the (hopefully) safer `sed 's/^'"$var"': //'`.

Let me know if it worked. It works on my system. (But then, the earlier version did, too.)
